### PR TITLE
Update porn.txt

### PR DESCRIPTION
**Pull request by LukeWhaskavsky**
## Summary

Added 11 porn sites.

### CODE REVIEW

File: porn.txt [hunk 1]
```diff
@@ -500299,3 +500299,14 @@
...
+0.0.0.0 www.filmikiporno.tv
+0.0.0.0 www.tube2017.com
+0.0.0.0 www.filmyporno.blog
+0.0.0.0 www.koloporno.com
+0.0.0.0 penisy.prv.pl
+0.0.0.0 pornelos.com
+0.0.0.0 pl.sex.com
+0.0.0.0 www.evooli.com
+0.0.0.0 www.pornohammer.com
+0.0.0.0 www.phica.eu/forums/
```
Comment (spirillen) [inline porn.txt:500311]: INVALID, not a domain

File: porn.txt [hunk 2]
```diff
@@ -500299,3 +500299,14 @@
 0.0.0.0 porngameshub.com
 0.0.0.0 pornhentai.games
 0.0.0.0 collegesexgames.com
+0.0.0.0 www.filmikiporno.tv
+0.0.0.0 www.tube2017.com
+0.0.0.0 www.filmyporno.blog
+0.0.0.0 www.koloporno.com
```
Comment (spirillen) [inline porn.txt:500305]: Not accessible

File: porn.txt [hunk 3]
```diff
@@ -500299,3 +500299,14 @@
 0.0.0.0 porngameshub.com
 0.0.0.0 pornhentai.games
 0.0.0.0 collegesexgames.com
+0.0.0.0 www.filmikiporno.tv
```
Comment (spirillen) [inline porn.txt:500302]: Not accessible

File: porn.txt (hunk 1 above)
Comment (LukeWhaskavsky) [reply porn.txt:500311]: Answers whois:
Domain Name: EVOOLI.COM
   Registry Domain ID: 2489954012_DOMAIN_COM-VRSN
   Registrar WHOIS Server: whois.namecheap.com
   Registrar URL: http://www.namecheap.com
   Updated Date: 2024-01-18T10:03:24Z
   Creation Date: 2020-02-08T11:08:23Z
   Registry Expiry Date: 2026-02-08T11:08:23Z
   
Domain Name: PORNOHAMMER.COM
   Registry Domain ID: 2010613051_DOMAIN_COM-VRSN
   Registrar WHOIS Server: whois.namecheap.com
   Registrar URL: http://www.namecheap.com
   Updated Date: 2024-02-12T07:23:51Z
   Creation Date: 2016-03-09T09:09:22Z
   Registry Expiry Date: 2025-03-09T09:09:22Z

Comment (LukeWhaskavsky) [reply porn.txt:500311]: Domain name: phica.eu
Status: Registered
Registered: 27 June 2022
Registrar: GoDaddy.com, LLC

File: porn.txt [hunk 4]
```diff
@@ -500299,3 +500299,14 @@
...
+0.0.0.0 www.tube2017.com
+0.0.0.0 www.filmyporno.blog
+0.0.0.0 www.koloporno.com
+0.0.0.0 penisy.prv.pl
+0.0.0.0 pornelos.com
+0.0.0.0 pl.sex.com
+0.0.0.0 www.evooli.com
+0.0.0.0 www.pornohammer.com
+0.0.0.0 www.phica.eu/forums/
+0.0.0.0 www.letmejerk.com
```
Comment (LukeWhaskavsky) [inline porn.txt:500312]: All 11 sites work for me.

File: porn.txt (hunk 1 above)
Comment (iam-py-test) [reply porn.txt:500311]: > `0.0.0.0 www.phica.eu/forums/`

I am sorry, but that is not a valid HOSTs entry; HOSTs files can only block entire domains, not specific URLs. It should be `0.0.0.0 www.phica.eu`.
Thank you

**Comment by gap579137** [comment]
Triage completed: add

Domain will be added to appropriate list(s) in next update.

**Pull request merged**